packages/firebase_ml_vision/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: firebase_ml_vision
 description: Flutter plugin for Firebase machine learning vision services.
 homepage: https://github.com/FirebaseExtended/flutterfire/tree/master/packages/firebase_ml_vision
-version: 0.11.0+2
+version: 0.12.0
```

Comment:
Versions are updated on release, could you revert this?

Reply:
Done.